guides/common/modules/proc_configuring-server-to-sync-content-using-exports.adoc

```diff
@@ -1,7 +1,7 @@
 [id="configuring-server-to-sync-content-using-exports_{context}"]
-= Configuring {Project} server to synchronize content using exports
+= Configuring {Project} server to synchronize content by using exports
 
-If you deployed your downstream {ProjectServer} as air-gapped, configure your {ProjectServer} as such to avoid attempts to consume content from a network.
+If you deployed your downstream {ProjectServer} as air gapped, configure your {ProjectServer} as such to avoid attempts to consume content from a network.
```

Comment:
By my search, "air-gapped" is more common than "air gapped". Do you have a source? If so, I'd be happy to provide a follow-up PR to unify this afterwards.

Reply:
"air-gapped" as an adjective and "air gapped" otherwise? Perhaps my reasoning was wrong, because now "as air-gapped" looks like an adverb. And I'm not sure if that's supposed to be hyphenated or not...

Reply:
@bangelic Can you please help us determine the correct hyphenation? :)

Reply:
In our existing docs, we use "air-gapped" and that seems to be the industry standard.

We use "air-gapped" in these files:
**common/modules/con_how-to-configure-inter-server-sync.adoc
common/modules/con_synchronizing-content-between-servers.adoc
common/modules/proc_configuring-server-to-sync-content-using-exports.adoc
common/modules/proc_updating-disconnected-server.adoc
common/modules/ref_system-requirements.adoc**

Reply:
I checked on our internal Slack with folks, and this should be correct way to hyphenate it:

- "An air-gapped system" (hyphenated)
- But "The system is air gapped" (not hyphenated)

According to IBM SG:

> Hyphenate compound adjectives and adverbs before a noun if readers might misinterpret the term or phrase.
> When a compound adjective follows a noun, you can omit the hyphen unless it is necessary to avoid ambiguity.